package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dao-v2-ui",
-  "version": "0.1.0",
+  "version": "2.2.0",
```

Comment:
What motivates skipping 2 majors?

Reply:
This version was never actually used. This is the dapp v2 so I major is now 2. Minor went from 1 to 2 for the changes.

Reply:
This does not comply with semver https://semver.org/.
major is not related to dao v2. minor should be reset if you update major. Have a look at this: https://semver.org/#spec-item-8

Reply:
Technically this change is a breaking change for a user, thus a major upgrade should be the correct choice. New version should be 1.0.0.

Reply:
Right, changed according to specs.

Reply:
Updated according to semver.